ogr/services/pagure/service.py

```diff
@@ -59,9 +59,17 @@ def __init__(
             self.session.mount("https://", adapter)
 
         self.header = {"Authorization": "token " + self._token} if self._token else {}
+        # By default ogr deals with Pagure API -> json
+        self.header["Accept"] = "application/json"
 
         if user_agent:
             self.header |= {"User-Agent": user_agent}
+        else:
+            try:
+                from ogr import __version__
+            except ImportError:
+                __version__ = "unknown"
```

Comment:
THis can probably happen in local git checkout, so maybe like this?

```suggestion
                __version__ = "dev"
```

But I would rather avoid overwriting `__version__`, let's add e.g. `ogr_version` variable and assign to it.